doc/nrf/protocols/wifi/sniffer_rx_operation.rst

```diff
@@ -0,0 +1,137 @@
+
+Monitor mode on the nRF70 device cannot operate concurrently with other modes, such as Station mode or SoftAP mode.
+
+you must configure the operating Wi-Fi channel on which the nRF70 device will monitor and receive 802.11 packets.
+When the device operates in Monitor mode, all 802.11 packets received on the configured channel will be sent up the stack by the nRF70 Series device and the Wi-Fi driver.
+
+To set the desired channel for 802.11 packet reception, use the ``NET_REQUEST_WIFI_CHANNEL`` runtime network management API.
+The channel configuration runtime NET-management API can be used to set the channel after setting the nRF70 Series device in Monitor mode.
+
+See the :ref:`wifi_shell_sample` sample for more information on configuring the mode and channel settings for raw packet reception through shell commands.
+
+The following table lists the shell commands and NET-management APIs that are used to enable Monitor and TX injection mode, and to select the desired Wi-Fi channel:
+
+.. list-table:: Wi-Fi packet reception network management APIs
+   :header-rows: 1
+
+   * - Network management APIs
+     - Wi-Fi shell command
+     - Example usage
+     - Description
+   * - net_mgmt(NET_REQUEST_WIFI_MODE)
+     - ``wifi mode -i<interface instance> <configuration>``
+     - ``wifi mode -i1 -m``
+     - Configure interface instance 1 to Monitor mode
+   * - net_mgmt(NET_REQUEST_WIFI_MODE)
+     - ``wifi mode -i<interface instance> <configuration>``
+     - ``wifi mode -i1 -s``
+     - Change interface instance 1 to Station mode
+   * - net_mgmt(NET_REQUEST_WIFI_CHANNEL)
+     - ``wifi channel -i<interface instance> <channel number to set>``
+     - ``wifi channel -i1 -c6``
+     - Set the raw transmission channel to ``6`` for interface 1
+
+.. _ug_nrf70_developing_monitor_mode_receive_packet_metadata:
+
+Monitor mode receive packet metadata
+************************************
+
+The IEEE 802.11 packet that is captured by the nRF70 Series device will prepend a proprietary header ``raw_rx_pkt_header``, which contains the following information about the received 802.11 packet:
+
+.. list-table:: Wi-Fi packet reception header elements
+   :header-rows: 1
+
+   * - Parameter name
+     - Type
+     - Description
+   * - frequency
+     - unsigned short
+     - Provides the current frequency at which the packet was received.
+       The frequency element can be used to compute the packet receive channel.
+   * - signal
+     - signed short
+     - Signal strength of the received packet.
+   * - rate_flags
+     - unsigned char
+     - Provides data on whether the received 802.11 packet is a legacy, HT, VHT or HE packet.
+   * - rate
+     - unsigned char
+     - Provides the data rate at which the packet was received.
+       It can be a ``legacy rate`` or an ``MCS rate`` based on the ``rate_flags`` value.
+
+You can refer to the relevant structure at:
+:file:`nrfxlib/nrf_wifi/fw_if/umac_if/inc/default/fmac_structs.h` - for the raw RX header
```

Comment:
Since there is only one entry so it would be better to present it in a single line rather than in list form..
```suggestion
You can refer to the relevant structure at :file:`nrfxlib/nrf_wifi/fw_if/umac_if/inc/default/fmac_structs.h` for the raw RX header.
```